coredb-operator/charts/coredb-operator/templates/cluster-role.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 rules:
   - apiGroups: ["coredb.io"]
     resources: ["coredbs", "coredbs/status"]
-    verbs: ["get", "list", "watch", "patch"]
+    verbs: ["create", "delete", "get", "list", "watch", "patch", "update"]
```

Comment:
As discussed, `create` & `delete` is not needed here, and `update` is only needed for `coredbs/status`. Approving it but please remove it after testing.